README.md

```diff
@@ -29,6 +29,128 @@ For available metrics please refer to the latest core's master branch
+
+  - a TimescaleDB instance running **on port 5432**. If the application the pipeline belongs to also wants a service running on 5432, then you need to change that.
+  - a Membrane pipeline with `membrane_timescaledb_reporter` set up to persist reports to the Timescale instance,
+  - this app
+
+### Setting up the Timescale instance
+
+This is a set of sample parameters, which should _just work_
+
+```bash
+docker run \
+-e POSTGRES_DB=membrane_timescaledb_reporter \
+-e POSTGRES_USER=postgres \
+-e POSTGRES_PASSWORD=postgres \
+--name dashboard-timescale \
+-d -p 5432:5432 \
+timescale/timescaledb:latest-pg14
+```
+
+### Wiring a membrane pipeline to a `membrane_timescaledb_reporter`
+
+This is the most complicated part of all of this. First, add `membrane_timescaledb_reporter` to your `mix.exs`:
+
+```elixir
+{:membrane_timescaledb_reporter, "~> 0.1.0"}
+```
+  
+Then, run
+
+```
+mix deps.get
+```
+
+After that, add the following lines to your `config/config.exs`
+
+```elixir
+config :membrane_core,
+  telemetry_flags: [
+    :links,
+    :inits_and_terminates,
+    {:metrics, [:caps]} # optional line, only add that if you want to
+	                # use some of the metrics
+  ]
+
+config :membrane_timescaledb_reporter, Membrane.Telemetry.TimescaleDB.Repo,
+  database: "membrane_timescaledb_reporter",
+  username: "postgres",
+  password: "postgres",
+  hostname: "localhost",
+  port: 5432,
+  chunk_time_interval: "3 minutes",
+  chunk_compress_policy_interval: "1 minute",
+  log: false
+
+config :membrane_timescaledb_reporter,
+  reporters: 5,
+  auto_migrate?: true
+```
+
+You can read about the metrics that can be specified in `telemetry_flags` in the [`Membrane.Telemetry` docs](https://hexdocs.pm/membrane_core/0.11.2/Membrane.Telemetry.html#module-enabling-certain-metrics-events).
+
+The next step is to forcibly recompile `membrane_core`:
+
+```
+mix deps.compile --force membrane_core
+```
+
+After you do this, the only thing left is to add `Membrane.Telemetry.TimescaleDB` to your supervision tree.  An example from Membrane Live:
+
+```elixir
+@impl true
+def start(_type, _args) do
+  children = [
+	Membrane.Telemetry.TimescaleDB, # this is the added line
+	MembraneLive.Repo,
+	{Phoenix.PubSub, name: MembraneLive.PubSub},
+	MembraneLiveWeb.Presence,
+	MembraneLiveWeb.Endpoint
+  ]
+
+  :ets.new(:presenters, [:public, :set, :named_table])
+  :ets.new(:presenting_requests, [:public, :set, :named_table])
+  opts = [strategy: :one_for_one, name: MembraneLive.Supervisor]
+  Supervisor.start_link(children, opts)
+end
+```
+
+After this step, your application should automatically log all configured events to the database. If you decided to use `docker-compose`, you don't need to do anything else. If you decided to set up everything by yourself, there's one more step left.
+
+### Running `membrane_dashboard`
+
+Fortunately, this is rather simple. You need to get all the dependencies:
```

Comment:
How about emphasizing that this one needs to be done in the directory where you have cloned the `membrane_dashboard`? I know it should be quite obvious, but the previous steps were done in the application directory :D